### PR TITLE
master-qa-52990

### PR DESCRIPTION
**Pull request by suilin**

### CODE REVIEW

**Comment by suilin** [comment]
ci:test:sf

**Comment by suilin** [comment]
ci:test

**Comment by xbrianlee** [comment]
ci:test

**Comment by xbrianlee** [comment]
ci:test:norebase

File: test.properties [hunk 1]
```diff
@@ -609,6 +609,31 @@
...
+    # Uncommment the following lines to deploy Elasticsearch 6.7.1
+    # instead of Elasticsearch 6.8.2.
+    #
+    #elasticsearch.ltr.version=1.1.1
+    #elasticsearch.ltr-linear-regression-model.zip.name=ltr-linear-regression-model.zip
+    #elasticsearch.ltr-linear-regression-model.zip.url=http://files.liferay.com/private/apps/elastic/elasticsearch/${elasticsearch.ltr-linear-regression-model.zip.name}
+    #elasticsearch.ltr.zip.name=ltr-${elasticsearch.ltr.version}-es${elasticsearch.version}.zip
+    #elasticsearch.ltr.url=http://es-learn-to-rank.labs.o19s.com/${elasticsearch.ltr.zip.name}
+
+    #elasticsearch.version=6.8.2
```
Comment (xbrianlee) [inline test.properties]: @suilin This needs to be changed to `#elasticsearch.version=6.7.1`. Sorry my mistake.

**Comment by xbrianlee** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Comment by suilin** [comment]
ci:test

**Pull request closed without merging**